File: C/section07/S07E05.c
#include <stdio.h>
#include <string.h>

int main() { 
    char login[100];
    char password[100];

    printf("Please enter your login: \n");
    gets(login);
        
    printf("Please enter your password: \n");
    gets(password);

    while(!strcmp(login, password)) {
        printf("\nYour password can't be the same as your login. \n");

        printf("Please enter your login: \n");
        gets(login);
        
        printf("Please enter your password: \n");
        gets(password);

    }

    printf("Login ok.");

    return 0;
}

File: Python/section07/S07e05.py
login = input("\nPlease enter your login: ")
password = input("Please enter your password: ")
if password == login:
    while login == password:
        print("\nYour password can't be identical than your password.\nPlease try again.")
        login = input("\nPlease enter your login: ")
        password = input("Please enter your password: ")
print("\nYou have succesfully logged in.")